BridgeAppSDK/SBAExternalIDOnboardingController.swift

```diff
@@ -53,22 +78,46 @@ public extension SBAExternalIDOnboardingController {
 
     public func registerUser() {
         
-        guard let text: String = self.registrationCodeTextField.text
+        // Check if the text string is valid
+        guard let text: String = self.registrationCodeTextField.text?.trim()
             where text.characters.count > 0 else {
-                let message = NSLocalizedString("Please enter a valid registration code.", comment: "Message for invalid registration code")
+                let message = Localization.localizedString("SBA_REGISTRATION_INVALID_CODE")
                 showAlertWithOk(nil, message: message, actionHandler: nil)
                 return
         }
         
+        // Check if this is a step view controller with verification
+        if let stepVC = self as? SBAExternalIDOnboardingControllerWithVerification,
```

Comment:
Swift Protocol extensions don't work that way. You can't override a function. It won't get called (only the base protocol implementation will be called.) My goal was to allow reverse-compatibility with Lilly so that Lilly doesn't have to implement methods it doesn't use (optional methods - which aren't allowed in swift)